File: 2) JAVA/areaFiguras.java
import java.util.Scanner;

public class Main {
	public static void main( String[ ] args ) {
		Scanner scanner = new Scanner( System.in );
		int a, b, c, trapezio, quadrado, triangulo;
		
		System.out.println( "Digite a medida A: " );
		a = scanner.nextInt( );
		
		System.out.println( "Digite a medida B: " );
		b = scanner.nextInt( );
		
		System.out.println( "Digite a medida C: " );
		c = scanner.nextInt( );
		
		quadrado = a * a;
		triangulo = ( a * b ) / 2;
		trapezio = ( a + b ) * c / 2;
		
		System.out.printf( "Area do quadrado: %d%nArea do triangulo: %d%nArea do trapezio: %d", quadrado, triangulo, trapezio );
		
		scanner.close( );
	}
}

File: 6) C E C++/areaFiguras.c
#include <stdio.h>

int main( ) {
	float a, b, c, quadrado, trapezio, triangulo;

	printf( "Digite o valor da medida A: " );
	scanf( "%f", &a );
	printf( "Digite o valor da medida B: " );
	scanf( "%f", &b );
	printf( "Digite o valor da medida C: " );
	scanf( "%f", &c );
	
	quadrado = a * a;
	trapezio = ( a + b ) * c / 2;
	triangulo = ( a * b ) / 2;
	
	printf( "Area quadrado:%.2f\nArea triangulo:%.2f\nArea trapezio:%.2f", quadrado, triangulo, trapezio );
}